deep_qa/data/tokenizers/word_and_character_tokenizer.py

```diff
@@ -81,10 +81,10 @@ def embed_input(self,
         # mask_split_axis argument to VectorMatrixSplit.
         words, characters = VectorMatrixSplit(split_axis=-1)(input_layer)
         word_embedding = embed_function(words,
-                                        embedding_name='word_' + embedding_name,
```

Comment:
Personally I think having the default embedding suffix being "embedding" is more clear, but it's not a big issue.